amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/MultiLangDaemonConfiguration.java

```diff
@@ -87,7 +87,7 @@ public void setWorkerId(String workerId) {
     @ConfigurationSettable(configurationClass = LeaseManagementConfig.class)
     private long failoverTimeMillis;
     @ConfigurationSettable(configurationClass = LeaseManagementConfig.class)
-    private int agedFailoverTimeMultiplier;
+    private Boolean doPriorityLeaseTaking;
```

Comment:
After doing some testing, I found that when this value is a primitive `boolean`, the config isn't set properly. This applies to all other boolean configs in multilang as well. Will make another PR fixing it for other configs